ASAManager/DeviceGroupsTests.cs

```diff
@@ -52,6 +52,8 @@ public async Task DeviceGroupsInitiallyWritten()
         /// Then checks if that device group is deleted,
         /// reference data will be updated again.
         /// This test takes 2+ minutes to run.
+        /// Keeping this test disabled because the latency to query device
```

Comment:
Is the intention to only run this test locally?

Reply:
For now yes, the test depended on timing which didn't hold up after the integration tests ran for a long time in one subscription. It's something we should revisit--it may not be possible to test this functionality in an integration test because we can't guarantee the latency of querying device groups.